source/_components/light.limitlessled.markdown

```diff
@@ -80,3 +80,129 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
+- The remaining bulbs are all E27 RGBW added to the relevant zone using the 2nd remote control from the left in the MiLight app v 3.7; this is the colour-only remote with 4 yellow group buttons at the bottom.
+  - Kitchen table, also added to zone 1, but from the colour only remote stated above. This appears to allow me to independently control it even though it's on the same zone as the 4x GU10 bulbs. This is because they're of a different 'type', 'white' vs 'rgbw' (see the config below). This is explained in [LimitlessLED example page](https://home-assistant.io/components/light.limitlessled/) 
+  - Lounge & Hall, both are bound to zone 2
+  - Study, bound to zone 3
+  - Bedroom, bound to zone 4
+
+Other Tips I found on an article on the LimitlessLED website that might be of use:
+
+We also had some feedback/tips from users:
+1. Tip#10. I had to use the LimitlessLED v1.7 app to connect the bridge to my LAN, then it worked, thanks.
+1. Tip#11. I had to use the correct remote screen in the app for my particular RGBW LEDs to sync, thanks.
+1. Tip#12. I had to clear remote code pairing using section 3.3.5 of the manual, and then the app and wifi
+bridge worked. Thx
+1. Tip#13. “I figured it out. I was using a fixture that held multiple bulbs. Apparently there is interference
+when you try to pair two bulbs at the same time. A fixture with a single bulb worked.”
+1. Tip#14. It worked after I changed my wifi router to the standard wifi secure settings WPA2/PSK AES.
+1. Tip#15. I was able to connect the bridge to my network after I set the SSID to visible.
+1. Tip#16. I placed the wifi bridge closer to the wifi router.
+1. Tip#17. After trying another usb power supply to the wifi bridge it worked.
+1. Tip#18. Might be some sneaky power leaking on to your socket with the switch even off. So remove
+the bulb from the socket, turn the switch on at the wall. Plug the bulb to the socket, and within 2
+seconds press the group1 on button on the remote. Dual white remote for dual white bulbs, RGBW
+remote for rgbw bulbs. Now for the App, you must sync it again separately, because the wifi bridge acts
+just like its own remote.
+
+
+### Add the config to the configuration.yaml file
+
+Here's my working light section:
+
+    light:
+      platform: limitlessled
+      bridges:
+        - host: 192.168.0.138
+          version: 6
+          port: 5987
+          groups:
+          - number: 1
+            type: white
+            name: Kitchen
+          - number: 1
+            type: bridge-led
+            name: Bridge Light
+          - number: 1
+            type: rgbw
+            name: Kitchen table
+          - number: 2
+            type: rgbw
+            name: Lounge & Hall
+          - number: 3
+            type: rgbw
+            name: Study
+          - number: 4
+            type: rgbw
+            name: Bedroom
+
+Then reboot your Pi.
+You should now see the lights on your home page in HomeAssistant.
+
+Things to note here...
+
```

Comment:
**Spelling & Grammar**:
_Note: This applies to the entire list below_
- Consider correcting the sequence numbers to actual number to improve the reading of the Markdown file.
- Start sentences with a capital letter.
- End sentences with a `.`.